corehq/apps/data_interfaces/views.py

```diff
@@ -518,6 +518,7 @@ def get_xform_ids(self, request):
             _request.couch_user.current_domain = self.domain
             _request.can_access_all_locations = request.couch_user.has_permission(self.domain,
                                                                                   'access_all_locations')
+            _request.always_allow_project_access = True
```

Comment:
This seems kinda magic, and possibly fragile. Does it override the permission check immediately above?

Reply:
Yeah it's totally magic and totally fragile. Specifically about the permissions check, "project access" just means allowed to access the project at all; it doesn't give blanket access to everything in the project.